src/main/java/Main.java

```diff
@@ -10,7 +10,7 @@
  */
 public class Main extends Application {
 
-    private Duke duke = new Duke();
+    private Duke duke = new Duke("../duke/data/duke.txt");
```

Comment:
Using relative address might be confusing - after gradle built the executable is inside ./build folder, however after release the .jar file could be anywhere. Probably it's better if you can move the .txt file into resource folder so that it's always stored at a fixed location.